### DISCUSSION TITLE
How to use RTK Query in ServerSideProps

### DISCUSSION THREAD

**Discussion by eitai** [reduxjs/redux-toolkit#2689] [Q&A]
hi i didnt find a straight forward explanation or example how to configure that .

this is how my store looks like today  if someone can help.

![image](https://user-images.githubusercontent.com/35004992/189138314-8ef719e7-0963-4c64-8776-78011889dc3e.png)

**Comment by phryneas** [comment]
What are you missing from https://redux-toolkit.js.org/rtk-query/usage/server-side-rendering#server-side-rendering-with-nextjs ?

**Reply by eitai** [reply]
how to configure it in the store and how to call it inside the SSR

**Reply by eitai** [reply]
![image](https://user-images.githubusercontent.com/35004992/189140098-3e958626-496c-4e88-a819-5b95440fa15d.png)

this is my api slice

**Reply by phryneas** [reply]
![image](https://user-images.githubusercontent.com/4282439/189141730-bc9379f4-93bc-42be-a43f-89be5b7adc91.png)

it seems you skipped the first point.

**Reply by eitai** [reply]
![image](https://user-images.githubusercontent.com/35004992/189143120-b616b038-c0f4-4368-a745-552a62f9d3f5.png)
 im not there it is, also i try to figure out where do i put all my reducers

**Reply by phryneas** [reply]
Your `makeStore` function is completely wrong for `next-redux-wrapper`. This will share information between all pages that were ever rendered server-side with every client and potentially expose private information.

**Reply by phryneas** [reply]
And after you have taken care of that... what is not clear about the second bullet point that literally tells you what you need to do in `getServerSideProps`?

**Reply by eitai** [reply]
i will try it again and back to you thanks
